spinoso-string/src/impls.rs

```diff
@@ -172,35 +172,35 @@ impl From<&str> for String {
 impl From<String> for Vec<u8> {
     #[inline]
     fn from(s: String) -> Self {
-        s.buf
+        s.inner.into_vec()
```

Comment:
I understand that this would consume `s`, which the previous code should also be doing right?

Reply:
yup that's right. `From` impls are consuming so we can destruct `s` to avoid an allocation.